### DISCUSSION TITLE
asus g16 keybord

### DISCUSSION THREAD

**Discussion by archfirein** [seerge/g-helper#3051] [General]
My laptop's keyboard goes to sleep when plugged in and I can't stop it and there needs to be an option for this in the app g helper

**Comment by seerge** [comment]
@archfirein it is difficult to understand what are you trying to say here. But sleep is a windows functionality, it has absolutely nothing to do with the app.

**Reply by archfirein** [reply]
My keyboard goes into sleep mode within 10 seconds and turns off its lights and this only happens when the application is open but does not happen when I close it.

**Reply by archfirein** [reply]
This problem occurs when the laptop is plugged in and the application is openin

**Reply by seerge** [reply]
@archfirein what exactly is "sleep mode" ? If you don't want backlight to turn off after inactivity - you can adjust that under `Extra` settings.

**Reply by archfirein** [reply]
In the settings, we can only select the option when running on battery power, but I cannot select it when plugged in and no matter what I do, it still turns off the keyboard lights.

**Reply by seerge** [reply]
That's not true, you can set separate setting for plugged in / on battery states. You need to set it to 0 to keep backlight permanently ON (i.e. turn timeout off)

![settings](https://github.com/user-attachments/assets/d074de84-9e5e-4480-bec2-9e5254fb2e5a)

**Comment by archfirein** [comment]
still doesn't fix the problem despite adjustments
![Screenshot 2024-09-02 150610](https://github.com/user-attachments/assets/31d39843-3d9b-4eb0-8070-674f680735c7)

**Reply by seerge** [reply]
@archfirein on your screenshot you have set backlight to turn off in **7 seconds.** 

As I have explained above - if you want to keep backlight on all the time - you need to set timeout to **0**

**Comment by archfirein** [comment]
Thank you for everything
